src/libs/registry/docset.cpp

```diff
@@ -824,5 +826,18 @@ static void sqliteScoreFunction(sqlite3_context *context, int argc, sqlite3_valu
         }
     }
 
-    sqlite3_result_int(context, score);
+    return score;
+}
+
+static void sqliteScoreFunction(sqlite3_context *context, int argc, sqlite3_value **argv)
+{
+    Q_UNUSED(argc);
+
+    sqlite3_result_int(
```

Comment:
This is ugly.

Just do casting first:

```cpp
const chat *needle = reinterpret_cast<const char *>(sqlite3_value_text(argv[0]));
const chat *haystack = reinterpret_cast<const char *>(sqlite3_value_text(argv[1]));
```

And you should get a single line.